dapps/marketplace/test/setup.js

```diff
@@ -3,20 +3,58 @@
  * the --file option to mocha. It sets up and tears down the infrastructure
  * (ethereum test node and IPFS) required to run tests.
  */
-
+import Web3 from 'web3'
+import bip39 from 'bip39'
+import hdkey from 'ethereumjs-wallet/hdkey'
 import services from './_services'
 
 const isWatchMode = process.argv.some(arg => arg === '-w' || arg === '--watch')
 
 let browser, webpackProcess, shutdownServices
 
+async function mnemonicToAddress(mnemonic) {
+  const seed = await bip39.mnemonicToSeed(mnemonic)
+  const masterKey = hdkey.fromMasterSeed(seed)
+  const masterWallet = masterKey.getWallet()
+  return masterWallet.getChecksumAddressString()
+}
+
 before(async function() {
   this.timeout(60000)
 
   shutdownServices = await services()
 
   browser = shutdownServices.extrasResult.browser
   webpackProcess = shutdownServices.extrasResult.webpackProcess
+
+  /**
+   * Need to fund the master account for the Relayer Purse
+   */
+  const web3 = new Web3('http://localhost:8545')
+  const netId = await web3.eth.net.getId()
+
+  if (parseInt(netId) === 999) {
+    if (!process.env.FORWARDER_MNEMONIC) {
+      throw new Error('FORWARDER_MNEMONIC missing, setup failed!')
+    }
+
+    const masterAddress = await mnemonicToAddress(process.env.FORWARDER_MNEMONIC)
```

Comment:
Wondering if we can move this code to the relayer and perhaps have an startup option 'fund=true' that can be called from @origin/services (similar to ganache inMemory option)

Reply:
That might be a good idea.  How was the previous `FUNDER_PK` account funded before?  I didn't see anything.

Reply:
I think I just used one of the default node accounts

Reply:
I agree though that it would be useful outside of testing(like local dev envs) and should probably go somewhere else but I don't know if this is the best option.  Could be an env var consumed by `app.js`, but then it has to give Relayer the option, which then should probably pass it to Purse, which then could fun during `init()`... for something that would only work in an environment with unlocked default accounts(ganache).  

What about including it in @origin/services whenever Relayer is enabled?  Would have to add web3 as a dependency there, but web3 is basically everywhere anyway.

Reply:
How about if we just do it as part of the `populate` script?

Reply:
That's a good spot.  Moved the funding to `populate.js`, let me know what you think.